src/fsharp/FSharp.Compiler.LanguageServer/LspTypes.fs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.LanguageServer
+
+// Types as defined at https://microsoft.github.io/language-server-protocol/specification
+
+/// Represents a zero-based line and column of a text document.
+type Position = {
```

Comment:
Nit: following the style guide might be helpful here as that's primarily what the rest of the codebase does: https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-record-declarations